enclave_manager/avalon_enclave_manager/graphene_wpe/graphene_wpe_enclave_info.py

```diff
@@ -14,30 +14,67 @@
 
 import logging
 
+from avalon_enclave_manager.enclave_attributes import EnclaveAttributes
 logger = logging.getLogger(__name__)
 
 # -------------------------------------------------------------------------
 
 
-class GrapheneWPESignupInfo():
+# class GrapheneWPESignupInfo(EnclaveAttributes):
+class GrapheneWPESignupInfo(EnclaveAttributes):
     """
     Signup Object that will be used by BaseEnclave Manager
     """
 
 # -------------------------------------------------------------------------
-    def __init__(self, worker_signup_json):
+    def __init__(self, config, worker_signup_json):
```

Comment:
Need to figure a way to common out with graphene_enclave_info.py. The codebase is pretty much same.

Reply:
Yes, refactoring needs to be looked at. Could be addressed as a separate PR.